### DISCUSSION TITLE
Some other observations

### DISCUSSION THREAD

**Discussion by Bumblebee001** [GrandOrgue/GoOdf#30] [General]
Image 2 for manuals does not work. It also gives an error on loading.

Text in labels: I would like to be able to enter a fresh line on pressing enter. Is text centred automatically?

**Comment by larspalo** [comment]
You need GO version 3.12+ for the new bitmaps. They are not backwards compatible and thus won't work in any older versions.

I'm pretty sure it's not possible to have a new line in the labels... It's really the same thing as what's written on the line Name= under the label entry. Perhaps with some special code characters... But it should certainly be possible to have multiple lines by setting the TextBreakWidth to a suitable value. Also, there are some newer bitmaps for the labels that will make any odf using them incompatible with older versions of GO.

**Comment by Bumblebee001** [comment]
Hi Lars,

For some strange reason the bass property of (ped/I) coupler reverts back to normal after reloading the organ.

**Reply by larspalo** [reply]
Yes, it's a bug! Thanks for reporting. I'll try to fix it soon, but I have too much things to do right now, so it might take a little while until I've created a new release. But I think you'll be happy to know that I've started working on the graphical display of a panel (that later might also include the possiblity to adjust positioning of its GUIElements directly on the rendered version of the panel instead of adjusting the x/y pixel values manually).

**Reply by Bumblebee001** [reply]
FANTASTIC!!!!!!! I love this software and have been using it since I got to know about it.... thankfully not long after you made it public. I am creating this new sample-set of an organ the recording of which happened to coincide with the development of your new and long awaited software.

I am at the moment working on a 4-5 rank mixture and I am anticipating difficulties with setting pitch information and setting Harmonic numbers. In the past, it was a hit or miss affair, correcting things to eliminate errors that propped up on loading in GO. But I really need to know if there is a proper way of doing this correctly from the start. Your advice would be most welcomed.
